examples(wms): Add minimal image and tile layer visualizations

ibgreen opened the pull request:
@jwasilgeo Adding your codepens to the gallery. 
- Want to repackage the single image into a CompositeLayer. 
- We should update these examples to work with actual WMS services.

jwasilgeo commented:
Quick draft attempt at changing the deck `TileLayer` codepen example to talk to a WMS:
https://codepen.io/jwasilgeo/pen/xxJjxox?editors=0010

There are some 4326 vs web merc reprojection visual errors that show up at around zoom levels 4 and smaller as one zooms out to the world.

This approach may also be more appropriate for a WMTS instead.

ibgreen commented:
@jwasilgeo Adopted your WMS example and added a single image WMS example as well. Moved the ImageServer examples to a different branch, focusing on WMS for now.

jwasilgeo commented:
Those examples are looking great, thanks @ibgreen!
